ci/38.yml

```diff
@@ -34,7 +34,7 @@ dependencies:
   - segregation
   - pointpats
   - giddy
-  - mapclassify
```

Comment:
Did you mean to remove `mapclassify` here?

Reply:
> Did you mean to remove `mapclassify` here?

yes, there was a duplicate of mapclassify in the spec.